README.md

```diff
@@ -13,3 +14,21 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+1. Constant factors that are disregarded during analysis can make a difference in actual performance between algorithms.
+2. The type of machine you are working with and the implementation can make a large difference. On different machines or languages, it may be faster or shorter.
+3. The input size also matters, especially with smaller inputs. With $O$, there is an input size that will mark the upper bound. Below that input size,
```

Comment:
Don't understand this -- what's the upper bound?

Reply:
Marking is the wrong word, with O(f(n)), after a certain input size, it is more efficient and faster than other algorithms. It 
 gives us the upper bound for run time for large enough inputs. But for inputs smaller than that n0, another algorithm may have a shorter run time.